app/controllers/home_controller.rb

```diff
@@ -52,12 +52,24 @@ def first_login_help
     render :action => "first_login_help", :layout => false
   end
 
+  def estimate_number(number)
+    digits=[Math.log10([number,1].max).to_i-3,0].max
+    divide=10 ** digits
+    divide*(number/divide).to_i
```

Comment:
Surrounding space missing for operator `*`.<br>Surrounding space missing for operator `/`.